### PR TITLE
FIX: correctly hides admin settings in channel settings

### PR DESCRIPTION
**Pull request by jjaffeux**
This would need even more tests which are being created in a separate branch.

### CODE REVIEW

File: plugins/chat/assets/javascripts/discourse/components/chat-channel-settings-view.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default class ChatChannelSettingsView extends Component {
   @computed("autoJoinAvailable", "togglingChannelWideMentionsAvailable")
   get adminSectionAvailable() {
     return (
-      this.chatGuardian.canEditChatChannel &&
```
Comment (nbianca) [inline plugins/chat/assets/javascripts/discourse/components/chat-channel-settings-view.js:64]: Can this method be converted into a property? I believe that's how we usually do it.

**Review by nbianca — APPROVED** [review]

**Review by pmusaraj — APPROVED** [review]
Looks good. Will the other branch/PR have tests for when the admin settings are present?

File: plugins/chat/assets/javascripts/discourse/components/chat-channel-settings-view.js (hunk 1 above)
Comment (jjaffeux) [reply plugins/chat/assets/javascripts/discourse/components/chat-channel-settings-view.js:64]: :+1: When I did there was some weird things going on with computed properties...

Anyways, chat-guardian on client side was an experiment and I don't think I will keep it now that we have system tests.

**Comment by jjaffeux** [comment]
> Looks good. Will the other branch/PR have tests for when the admin settings are present?

Yes that's the goal indeed

**Pull request merged**